### PR TITLE
[Windows] Support all possible suffixes for console wrapper

### PR DESCRIPTION
**Pull request by AThousandShips**
The official builds use `_console.exe` but the custom built version use `.console.exe`, this adds support for both, otherwise you can't export with the wrapper automatically added if you use a custom build directly, a bit hacky perhaps but I think it's better than changing the build format

Edit: Added support for all the suffixes supported by the console wrapper itself

### CODE REVIEW

File: editor/export/editor_export_platform_pc.cpp [hunk 1]
```diff
@@ -151,15 +151,20 @@ Error EditorExportPlatformPC::prepare_template(const Ref<EditorExportPreset> &p_
...
 	Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 	da->make_dir_recursive(p_path.get_base_dir());
 	Error err = da->copy(template_path, p_path, get_chmod_flags());
-	if (err == OK && copy_wrapper && FileAccess::exists(wrapper_template_path)) {
-		err = da->copy(wrapper_template_path, p_path.get_basename() + ".console.exe", get_chmod_flags());
+	if (err == OK && copy_wrapper) {
+		if (FileAccess::exists(wrapper_template_path_underscore)) {
+			err = da->copy(wrapper_template_path_underscore, p_path.get_basename() + ".console.exe", get_chmod_flags());
+		} else if (FileAccess::exists(wrapper_template_path_period)) {
+			err = da->copy(wrapper_template_path_period, p_path.get_basename() + ".console.exe", get_chmod_flags());
```
Comment (AThousandShips) [inline editor/export/editor_export_platform_pc.cpp:166]:
```suggestion
			err = da->copy(wrapper_template_path_underscore, wrapper_out_path, get_chmod_flags());
		} else if (FileAccess::exists(wrapper_template_path_period)) {
			err = da->copy(wrapper_template_path_period, wrapper_out_path, get_chmod_flags());
```

**Comment by bruvzg** [comment]
For the reference, the wrapper itself also supports space and no separator.

https://github.com/godotengine/godot/blob/dc91479082bd9cd56cd74282573492fe1ba9618a/platform/windows/console_wrapper_windows.cpp#L72

**Comment by AThousandShips** [comment]
Will add a list of cases instead and include all four and do a simple loop over them

File: editor/export/editor_export_platform_pc.cpp [hunk 2]
```diff
@@ -151,15 +151,27 @@ Error EditorExportPlatformPC::prepare_template(const Ref<EditorExportPreset> &p_
...
 
 	Ref<DirAccess> da = DirAccess::create(DirAccess::ACCESS_FILESYSTEM);
 	da->make_dir_recursive(p_path.get_base_dir());
 	Error err = da->copy(template_path, p_path, get_chmod_flags());
-	if (err == OK && copy_wrapper && FileAccess::exists(wrapper_template_path)) {
-		err = da->copy(wrapper_template_path, p_path.get_basename() + ".console.exe", get_chmod_flags());
+	if (err == OK && copy_wrapper) {
+		for (int i = 0; wrapper_extensions[i]; ++i) {
+			if (FileAccess::exists(template_path.get_basename() + wrapper_extensions[i])) {
+				err = da->copy(template_path.get_basename() + wrapper_extensions[i], p_path.get_basename() + ".console.exe", get_chmod_flags());
```
Comment (AThousandShips) [inline editor/export/editor_export_platform_pc.cpp:171]:
```suggestion
			const String wrapper_path = template_path.get_basename() + wrapper_extensions[i];
			if (FileAccess::exists(wrapper_path)) {
				err = da->copy(wrapper_path, p_path.get_basename() + ".console.exe", get_chmod_flags());
```
Will push once CI has finished

**Review by bruvzg — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by AThousandShips** [comment]
Thank *you*!

**Pull request merged**